packages/splits-sdk/src/constants/index.ts

```diff
@@ -111,7 +112,11 @@ export const FANTOM_CHAIN_IDS = [ChainId.FANTOM]
 export const AVALANCHE_CHAIN_IDS = [ChainId.AVALANCHE]
 export const BSC_CHAIN_IDS = [ChainId.BSC]
 export const AURORA_CHAIN_IDS = [ChainId.AURORA]
-export const ZORA_CHAIN_IDS = [ChainId.ZORA, ChainId.ZORA_GOERLI]
+export const ZORA_CHAIN_IDS = [
+  ChainId.ZORA,
+  ChainId.ZORA_GOERLI,
+  ChainId.ZORA_SEPOLIA,
+]
```

Comment:
I think we need a few more changes, unless you deployed more contracts onto Zora Sepolia?

Right now the SDK assumes all networks contain splits, waterfall, vesting, and recoup contracts. You can see for example down below we have: `const WATERFALL_CHAIN_IDS = ALL_CHAIN_IDS.slice()`

If waterfall is not on zora sepolia, we'll need to update that line (and the other relevant ones like it) to not include that chain.